### DISCUSSION TITLE
SS Leaderboard/some sort of option per map

### DISCUSSION THREAD

**Discussion by Mijii1999** [ppy/osu#26850] [Ideas]
Adding a new leaderboard (or some sort of toggle/filter/whatever) apart from the currently existing ones(aka Global, Friends, Country, Selected Mods) that shows only scores with 100.00% accuracy. While this might sound like a niche feauture(and tbf it kinda is) I think it would be cool because there's a part of the community that focuses on SS grinding. In the future when DT rate changes get ranked LBs  will look all over the place(getting ur 1.7x speed SS sniped by a low 98.3% 1.8x speed isn't cool, specially when it's potentially less pp). **The ranking position would be based on the score multiplier of the plays** (so at the top of the leaderboard there would be the HDDTHRFL SS plays with 2x DT speed without Classic mod and at the very bottom the scores with the lowest score multipliers, which AFAIK would be EZNFHTSO with Classic mod) ---> ofc the LB would be limited to the top 50 best scores (or maybe 100? , 100 score LBs is something I would personally love). Just to be clear this would just be an "alternative" leaderboard (like the Selected Mods one), this idea is not trying to replace the "Global" leaderboard as the default.
Edit: Not to mention that this could potentially open the door for a new competitive scene in the game, which is always a good thing.

**Comment by Sleepteiner** [comment]
I think what would be even better than this is an accuracy sorted leaderboard option. It could offer the same benefit as your proposal, but also finally offer accuracy-focused players a way to compete on leaderboards without having to worry about combo.

**Reply by Rekunan** [reply]
To also talk about the sniped play possibly being lower pp, pp sort is an option that is planned, so at least that part is def going to be fixed.
